src/estimator_example.cpp

```diff
@@ -80,11 +82,28 @@ void estimator_example::estimate(const params_s &params, const input_s &input, o
     float qhat = lpf_gyro_y;
     float rhat = lpf_gyro_z;
 
-    // low pass filter static pressure sensor and invert to esimate altitude
+    // low pass filter static pressure sensor
     //lpf_static = alpha1*lpf_static + (1-alpha1)*input.static_pres;
-    //float hhat = lpf_static/params.rho/params.gravity;
```

Comment:
why didn't you just uncomment these lines? I think you have lots of redundancy in the code with your additions.